dmn-engine/src/main/scala/org/camunda/dmn/evaluation/InvocationEvaluator.scala

```diff
@@ -24,15 +12,12 @@ class InvocationEvaluator(
   def eval(invocation: ParsedInvocation,
            context: EvalContext): Either[Failure, Val] = {
 
-    evalParameters(invocation.bindings, context).right.flatMap { p =>
+    evalParameters(invocation.bindings, context).flatMap { p =>
       val ctx = context.copy(variables = context.variables ++ p.toMap)
 
-      evalBkm(invocation.invocation, ctx).right
-        .map { result =>
-          context.audit(invocation, SingleEvaluationResult(result))
-
-          result
-        }
+      val result = evalBkm(invocation.invocation, ctx)
+      context.audit(invocation, result)
+      result
```

Comment:
Nitpick, we could align the structure here with the other evaluators.

So, it could look like this:
``` 
  def eval(invocation: ParsedInvocation,
           context: EvalContext): Either[Failure, Val] = {

    val result = evalParameters(invocation.bindings, context).flatMap { p =>

      val ctx = context.copy(variables = context.variables ++ p.toMap)
      evalBkm(invocation.invocation, ctx)
    }

    context.audit(invocation, result)
    result
  }
```